<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>912</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="key_Refresh_COM">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>350</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>刷新串口</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_serialport_onoff">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>打开串口</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="text_tx">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>310</y>
      <width>311</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="text_rx">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>40</y>
      <width>311</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="set_com">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="set_baudrate">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>80</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 9pt &quot;微软雅黑&quot;;</string>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
    <property name="currentText">
     <string>115200</string>
    </property>
    <item>
     <property name="text">
      <string>300</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2400</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4800</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>14400</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>19200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>38400</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>56000</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>115200</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>128000</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>256000</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>460800</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>512000</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>921600</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1500000</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="btn_enter_tx">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>350</y>
      <width>112</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>发送</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_clear_tx">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>310</y>
      <width>112</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>清除发送</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_clear_rx">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>190</y>
      <width>112</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>清除接收</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="data_0_0">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>10</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="data_0_1">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>10</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="data_0_3">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>10</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="data_0_2">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>10</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="data_1_2">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>80</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="data_1_3">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>80</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="data_1_1">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>80</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="data_1_0">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>80</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="data_2_2">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>150</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="data_2_3">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>150</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="data_2_1">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>150</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="data_2_0">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>150</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="data_3_2">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>220</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="data_3_3">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>220</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="data_3_1">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>220</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="data_3_0">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>220</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文行楷</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>端口名</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文行楷</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>波特率</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文行楷</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>数据位</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文行楷</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>校验位</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文行楷</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>停止位</string>
    </property>
   </widget>
   <widget class="QComboBox" name="set_databits">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>150</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <property name="currentText">
     <string>8</string>
    </property>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="set_parity">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>220</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>None</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Even</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Mark</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Odd</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="set_stopbits">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>290</y>
      <width>120</width>
      <height>40</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1.5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>0</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文行楷</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>接收区</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>16</width>
      <height>371</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_stop_rx">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>230</y>
      <width>112</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>停止接收</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_set_save_path">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>270</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>文件路径</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_save_data">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>270</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>保存文件</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_status">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>300</y>
      <width>291</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>270</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文行楷</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>发送区</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>912</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
